js/index.js

```diff
@@ -20,7 +20,7 @@ INSTRUCTIONS:
     project, when tests are failing, 
     you should get helpful errors 
     along the way!
-    ************/
+    **************/ 
 
```

Comment:
I suggest you make a more descriptive commit message. [This article could help you with that.](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something like Add page title to index.html